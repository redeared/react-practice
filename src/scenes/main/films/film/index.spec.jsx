import React from 'react';
import { create } from 'react-test-renderer';

import { Film } from './index';

describe('Film', () => {
    it('should render without crash', () => {
        const comp = create(<Film value={{ title: 'title', date: 'date' }} />).toJSON();

        expect(comp).toMatchSnapshot();
    });
});
