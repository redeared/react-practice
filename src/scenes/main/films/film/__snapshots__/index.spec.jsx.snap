// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film should render without crash 1`] = `
Array [
  <img
    alt="title"
    className="img"
    src=""
  />,
  <div>
    TITLE
      
    <span>
      date
    </span>
  </div>,
  <div />,
]
`;
